src/app/@core/services/user.service.ts

```diff
@@ -26,7 +26,6 @@ export class UserService {
     this.http.get(this.getUserUrl + username, httpOptions).subscribe(
```

Comment:
use header of httpService

Reply:
imported httpService to userService to use